src/olympia/blocklist/models.py

```diff
@@ -569,9 +572,13 @@ class KintoImport(ModelBase):
         OUTCOME_REGEXBLOCKS: 'Added blocks from regex',
         OUTCOME_NOMATCH: 'No matches',
     }
-    kinto_id = models.CharField(
-        unique=True, max_length=255, null=False, default='')
+    legacy_id = models.CharField(
+        unique=True, max_length=255, null=False, default='',
+        db_column='kinto_id')
```

Comment:
So the column in the db remains `kinto_id`?

Reply:
yes.  The ordering means none of the migrations actually cause any SQL* be generated so the database is the same as before (zero downtime/impact during the push).  Without the migration file though django will continue to think/complain the database isn't up to date vs the modesl.

*this is what will be run in reality, from `./manage.py sqlmigrate blocklist 0015`:
```
BEGIN;
--
-- Rename table for KintoImport to blocklist_kintoimport
--
--
-- Rename model KintoImport to LegacyImport
--
--
-- Alter field kinto_id on block
--
--
-- Rename field kinto_id on block to legacy_id
--
COMMIT;
```
(it's all comments)